static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx

```diff
@@ -0,0 +1,138 @@
+import {Step, StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import type {DocsParams} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {useSourcePackageRegistries} from 'sentry/components/onboarding/gettingStartedDoc/useSourcePackageRegistries';
+import {useUrlPlatformOptions} from 'sentry/components/onboarding/platformOptionsControl';
+import ConfigStore from 'sentry/stores/configStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {space} from 'sentry/styles/space';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function FeatureFlagOnboardingLayout({
+  docsConfig,
+  dsn,
+  platformKey,
+  projectId,
+  projectSlug,
+  newOrg,
+  projectKeyId,
+  configType = 'onboarding',
+  integration = '',
+  provider = '',
+}: OnboardingLayoutProps) {
+  const api = useApi();
+  const organization = useOrganization();
+  const {isPending: isLoadingRegistry, data: registryData} =
+    useSourcePackageRegistries(organization);
+  const selectedOptions = useUrlPlatformOptions(docsConfig.platformOptions);
+  const {isSelfHosted, urlPrefix} = useLegacyStore(ConfigStore);
+
+  const {introduction, steps} = useMemo(() => {
+    const doc = docsConfig[configType] ?? docsConfig.onboarding;
+
+    const docParams: DocsParams<any> = {
+      api,
+      projectKeyId,
+      dsn,
+      organization,
+      platformKey,
+      projectId,
+      projectSlug,
+      isFeedbackSelected: false,
+      isPerformanceSelected: false,
+      isProfilingSelected: false,
+      isReplaySelected: false,
+      sourcePackageRegistries: {
+        isLoading: isLoadingRegistry,
+        data: registryData,
+      },
+      platformOptions: selectedOptions,
+      newOrg,
+      isSelfHosted,
+      urlPrefix,
+      integration,
+    };
+
+    return {
+      introduction: doc.introduction?.(docParams),
+      steps: [
+        ...doc.install(docParams),
+        ...doc.configure(docParams),
+        ...doc.verify(docParams),
+      ],
+      nextSteps: doc.nextSteps?.(docParams) || [],
+    };
+  }, [
+    docsConfig,
+    dsn,
+    isLoadingRegistry,
+    newOrg,
+    organization,
+    platformKey,
+    projectId,
+    projectSlug,
+    registryData,
+    selectedOptions,
+    configType,
+    urlPrefix,
+    isSelfHosted,
+    api,
+    projectKeyId,
+    integration,
+  ]);
+
+  return (
+    <AuthTokenGeneratorProvider projectSlug={projectSlug}>
+      <Wrapper>
+        {introduction && <Introduction>{introduction}</Introduction>}
+        <Steps>
+          {steps.map(step =>
+            step.type === StepType.CONFIGURE ? (
+              <Step
+                key={step.title ?? step.type}
+                {...{
+                  ...step,
+                }}
+              />
+            ) : (
+              <Step key={step.title ?? step.type} {...step} />
+            )
+          )}
```

Comment:
I don't see a difference in those branches 🤔  + you are double spreading `step`
```suggestion
          {steps.map(step => (
              <Step key={step.title ?? step.type} {...step} />
          )}
```

Reply:
ah sorry, that was copied over from the replay onboarding but i deleted the change in the first branch🤦🏻‍♀️ good catch